driver-core/src/main/com/mongodb/LoggerSettings.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb;
+
+import com.mongodb.annotations.Immutable;
+
+import java.util.Objects;
+
+import static com.mongodb.assertions.Assertions.notNull;
+
+/**
+ * An immutable class representing settings for logging.
+ *
+ * <p>
+ * The driver logs using SLF4J 1.0 API with a root logger of {@code org.mongodb.driver}.
+ * </p>
+ *
+ * @since 4.9
+ */
+@Immutable
+public final class LoggerSettings {
```

Comment:
[optional]
This documentation seems to be a good place to leave a link to https://www.mongodb.com/docs/drivers/java/sync/current/fundamentals/logging.

Reply:
Done

Reply:
Note: I couldn't use the existing javadoc taglets due to the common prefix that they use.  So just did it the old-fashioned way (which I like better anyway because it will show up in IntelliJ more clearly)